Formula/kong.rb

```diff
@@ -3,26 +3,19 @@ class Kong < Formula
   homepage "https://docs.konghq.com"
 
   stable do
-    url "https://bintray.com/kong/kong-src/download_file?file_path=kong-2.1.0.tar.gz"
-    sha256 "8e7ceae26a2755225e085f4b95a6972778d339dd877d3d2165a672117e6caf69"
```

Comment:
Is there a reason we no longer includes the checksum of the tarball?

Reply:
two reasons:
- I don't believe from an infosec POV it provides anything of value. Bintray is SSL and the assets are signed
- Updating that SHA will be hard 😆 

Instead of a manually updated SHA what if we used the `asc` that bintray calculates for all our assets (though again feels a little like security theatre)

Reply:
AFAIK the `sha256` checksum is a common practice among official formulas (e.g. all the ones under https://github.com/Homebrew/homebrew-core/tree/master/Formula) even when they came from supposedly "credible sources" such as `https://mirrors.kernel.org`. Bintray is HTTPS but that does it may be breached. Since we are already updating the filenames manually, I don't see much work associated with updating the `sha256` value at the same time. It is definitely providing more value than just a security theater IMHO.

Reply:
I don't agree but will update